src/frontend/src/lib/components/transactions/Transaction.svelte

```diff
@@ -31,7 +32,7 @@
 		<span class="inline-block first-letter:capitalize"><slot /></span>
 
 		<div slot="icon">
-			{#if nonNullish(token)}
+			{#if nonNullish(token) && !hideTokenLogo}
```

Comment:
The token is mandatory now, I guess we can remove `nonNullish(token)`?

Reply:
make sense, done!